utilities/src/main/java/org/craftercms/commons/aws/AwsUtils.java

```diff
@@ -15,19 +15,20 @@
  */
 package org.craftercms.commons.aws;
 
-import com.amazonaws.client.builder.ExecutorFactory;
-import com.amazonaws.services.s3.AmazonS3;
-import com.amazonaws.services.s3.model.CopyObjectRequest;
-import com.amazonaws.services.s3.transfer.Transfer;
-import com.amazonaws.services.s3.transfer.TransferManager;
-import com.amazonaws.services.s3.transfer.TransferManagerBuilder;
-import com.amazonaws.services.s3.transfer.internal.TransferStateChangeListener;
+import software.amazon.awssdk.services.s3.S3AsyncClient;
+import software.amazon.awssdk.services.s3.model.CopyObjectRequest;
+import software.amazon.awssdk.transfer.s3.S3TransferManager;
```

Comment:
Now that we have the new transfer manager, should we add a method that allows copying directly from one bucket to another bucket, to speed up publishing? If I recollect I was told we could do that in V2 of the AWS library @phuongnq @jmendeza

Reply:
Conversation will be moved to another ticket, not necessary right now